pathway-analysis/qusage_single_dataset.Rmd

```diff
@@ -0,0 +1,420 @@
+---
+title: "Pathway analysis with QuSAGE: Single dataset"
+output: 
+  html_notebook:
+    toc: TRUE
+    toc_float: TRUE
+author: J. Taroni for ALSF CCDL
+date: 2019
+---
+
+## Background
+
+In this module, we'll demonstrate how to perform pathway analysis using
+Quantitative Set Analysis of Gene Expression (QuSAGE) 
+([Yaari et al. _NAR_. 2013.](https://doi.org/10.1093/nar/gkt660)).
+QuSAGE, implemented in the [`qusage` bioconductor package](https://bioconductor.org/packages/release/bioc/html/qusage.html),
+has some nice features:
+
+* It takes into account inter-gene correlation (a source of type I error).
+* It returns more information than just a p-value. 
+That's useful for analyses you might want to perform downstream.
+* Built-in visualization functionality.
+
+We recommend taking a look at the original publication (Yaari et al.) and 
+the R package documentation to learn more.
+
+## Gene sets
+
+Here we'll work with a mouse dataset.
```

Comment:
This sentence feels a bit abrupt and maybe out of place. Maybe put it in the next paragraph?